inaugurator/consts.py

```diff
@@ -0,0 +1,4 @@
+import os
+
+INAUGURATOR_LOG_FILE_NAME = "/inaugurator.log"
```

Comment:
Actually, it seems like the shutil.copy wouldn't work because of it, since `os.path.join('/destRoot', '/inaugurator.log') == '/inaugurator.log'`.